frameworks/desktop/views/split.js

```diff
@@ -561,7 +561,7 @@ SC.SplitView = SC.View.extend({
       if (isResizable === useResizable) {
         // if outOfSize === -1 then we are aggressively resizing (not resizing proportionally)
         if (outOfSize === -1) size += diff;
-        else size += (size / outOfSize) * diff;
+        else if (outOfSize !== 0) size += (size / outOfSize) * diff;
```

Comment:
Just wondering whether it wouldn't be easier to provide a default:

`else size += (size / (outOfSize || 1)) * diff;`

because not only 0 is a problem, but essentially every falsy value. As `outOfSize` is a function parameter, it is possible it is initialized to something non numeric.